examples/nextjs-lexical-ai/app/globals.css

```diff
@@ -0,0 +1,196 @@
+    --accent-foreground: 0 0% 98%;
+    --destructive: 0 62.8% 30.6%;
+    --destructive-foreground: 0 85.7% 97.3%;
+    --border: 240 3.7% 15.9%;
+    --input: 240 3.7% 15.9%;
+    --ring: 240 4.9% 83.9%;
+  }
+}
+
+::selection {
+  @apply bg-indigo-100/75;
+}
+
+p {
+  margin: 0.8rem 0;
+}
+
+h1 {
+  font-size: 2rem;
+  font-weight: 800;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+h3 {
+  font-size: 1.25rem;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+h4 {
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+h5 {
+  font-size: 0.875rem;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+h6 {
+  font-size: 0.875rem;
+  font-weight: 500;
+  line-height: 1.2;
+}
+
+blockquote {
+  box-sizing: border-box;
+  border-left: 3px solid rgb(90, 88, 99);
+  padding-left: 0.9rem;
+  margin: 0.8rem 0;
+}
+
+ul {
+  display: block;
+  list-style-type: disc;
+  margin-block-start: 1rem;
+  margin-block-end: 1rem;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 40px;
+  unicode-bidi: isolate;
+}
+
+ol {
+  display: block;
+  list-style-type: decimal;
+  margin-block-start: 1rem;
+  margin-block-end: 1rem;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding-inline-start: 40px;
+  unicode-bidi: isolate;
+}
+
+li {
+  display: list-item;
+  text-align: -webkit-match-parent;
+  unicode-bidi: isolate;
+}
+
+li::marker {
+  unicode-bidi: isolate;
+  font-variant-numeric: tabular-nums;
+  text-transform: none;
+  text-indent: 0 !important;
+  text-align: start !important;
+  text-align-last: start !important;
+}
+
+code {
+  background: rgb(241, 241, 244);
+  padding: 1px;
+  font-size: 0.94em;
+  border-radius: 4px;
+}
+
+select {
+  appearance: none;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 20 20'%3e%3cpath stroke='%236b7280' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='M6 8l4 4 4-4'/%3e%3c/svg%3e");
+  background-position: right 0.3rem center;
+  background-repeat: no-repeat;
+  background-size: 1.5em 1.5em;
+  padding-right: 1.8rem;
+}
+
+.lexical-underline {
+  text-decoration: underline;
+}
+
+.lexical-strikethrough {
+  text-decoration: line-through;
+}
+
+.lb-root {
+  --lb-accent: rgb(84, 85, 250);
+  --lb-elevation-shadow: rgba(0, 0, 0, 0) 0px 0px 0px 0px, rgba(0, 0, 0, 0) 0px 0px 0px 0px, rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.1) 0px 4px 6px -4px;
+}
+
+.lb-lexical-floating-composer, .lb-lexical-floating-threads {
+  border: 1px solid rgba(228, 228, 231, 0.8);
+}
+
+.lb-thread .lb-composer {
+  border-top: 1px solid rgba(228, 228, 231, 0.8);
+}
+
+.lb-composer, .lb-thread {
+  font-size: 14px;
+}
+
+.lb-thread-composer:after {
+  content: none;
+}
+
+.lb-thread[data-state="active"],
+.lb-thread[data-state="active"] .lb-comment,
+.lb-thread[data-state="active"] .lb-composer {
+  background: var(--lb-accent-subtle);
```

Comment:
Do you think that's needed with the X position change and the increased shadow? imo it's a bit too strong.